.gitmodules

```diff
@@ -58,3 +55,24 @@
 [submodule "scripts/packages/IJuliaCore"]
 	path = scripts/packages/IJuliaCore
 	url = https://github.com/JuliaLang/IJuliaCore.jl.git
+[submodule "scripts/packages/Crayons"]
+	path = scripts/packages/Crayons
+	url = https://github.com/KristofferC/Crayons.jl.git
+[submodule "scripts/packages/DataAPI"]
```

Comment:
I don't think we need that? At least I wasn't able to figure out which package might be pulling this in?

Reply:
JSON, but we have an older version that doesn't need it.

Reply:
Yes, we are on an older version of JSON because it doesn't pull in anything. But even `JSON#master` doesn't seem to pull this in? I think it pulls in Parsers.jl? But in any case, I guess we can remove, right?

Reply:
Right, it's a transitive dependency. I was mostly just looking at https://juliahub.com/ui/Packages/JuliaFormatter/OWZnm/0.15.2?t=1 and mentally filtering out what I knew were dependencies of JSON, but missed this one.